server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -321,60 +325,58 @@ Issues related to authorization may exist just for some other versions of {jdgse
 But this option usually should be omitted in production environments to avoid the ability for remote access to your server. For more information,
 see the link:{appserver_socket_link}[_{appserver_socket_name}_].
 
-. Start server `jdg1`:
+. Start server `server1`:
 +
 [source,subs="+quotes"]
 ----
-cd JDG1_HOME/bin
+cd SERVER1_HOME/bin
 ./standalone.sh -c clustered.xml -Djava.net.preferIPv4Stack=true \
   -Djboss.default.multicast.address=234.56.78.99 \
-  -Djboss.node.name=jdg1 -b _PUBLIC_IP_ADDRESS_
+  -Djboss.node.name=server1 -b _PUBLIC_IP_ADDRESS_
 ----
 +
 
-. Start server `jdg2`. There is a different multicast address, so the `jdg1` and `jdg2` servers are not directly clustered with each other; rather, they are just connected through the RELAY2 protocol, and the TCP JGroups stack is used for communication between them. The start up command looks like this:
+. Start server `server2`. There is a different multicast address, so the `server1` and `server2` servers are not directly clustered with each other; rather, they are just connected through the RELAY2 protocol, and the TCP JGroups stack is used for communication between them. The start up command looks like this:
 +
 [source,subs="+quotes"]
 ----
-cd JDG2_HOME/bin
+cd SERVER2_HOME/bin
 ./standalone.sh -c clustered.xml -Djava.net.preferIPv4Stack=true \
   -Djboss.default.multicast.address=234.56.78.100 \
-  -Djboss.node.name=jdg2 -b _PUBLIC_IP_ADDRESS_
+  -Djboss.node.name=server2 -b _PUBLIC_IP_ADDRESS_
 ----
 +
 
-. To verify that channel works at this point, you may need to use JConsole and connect either to the running `JDG1` or the `JDG2` server. When you use the MBean `jgroups:type=protocol,cluster="cluster",protocol=RELAY2` and operation `printRoutes`, you should see output like this:
+. To verify that channel works at this point, you may need to use JConsole and connect either to the running `SERVER1` or the `SERVER2` server. When you use the MBean `jgroups:type=protocol,cluster="cluster",protocol=RELAY2` and operation `printRoutes`, you should see output like this:
 +
 ```
-site1 --> _jdg1:site1
-site2 --> _jdg2:site2
+site1 --> _server1:site1
+site2 --> _server2:site2
 ```
 When you use the MBean `jgroups:type=protocol,cluster="cluster",protocol=GMS`, you should see that the attribute member contains just single member:
-.. On `JDG1` it should be like this:
+.. On `SERVER1` it should be like this:
 +
 ```
-(1) jdg1
+(1) server1
 ```
 +
-.. And on JDG2 like this:
+.. And on SERVER2 like this:
 +
 ```
-(1) jdg2
+(1) server2
 ```
 +
 NOTE: In production, you can have more {jdgserver_name} servers in every data center. You just need to ensure that {jdgserver_name} servers in same data center are using the same multicast address (In other words, the same `jboss.default.multicast.address` during startup). Then in jconsole in `GMS` protocol view, you will see all the members of current cluster.
```

Comment:
`jboss.default.multicast.address` => `jgroups.mcast_addr`
`Then in jconsole in `GMS` protocol view, you will see all the members of current cluster.` => once again, I would say to check the logs for `ISPN000094: Received new cluster view for channel cluster: ...`